tests/chebyshev_test.py

```diff
@@ -0,0 +1,795 @@
+import numpy as np
+from numpy.polynomial import chebyshev as ncs
+import numpy.polynomial.polyutils as npu
+from numpy.polynomial.polyutils import RankWarning
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from jax import numpy as jnp
+from jax._src import test_util as jtu
+import jax.numpy.polynomials as pu
+from jax.numpy.polynomials import chebyshev as cs
+
+from jax.config import config
+config.parse_flags_with_absl()
+
+if config.jax_enable_x64:
+  all_dtypes = ["int64", "float64", "complex128"]
+else:
+  all_dtypes = ["int32", "float32", "complex64"]
+
+class TestChebyshev(jtu.JaxTestCase):
+
+  @parameterized.named_parameters(jtu.cases_from_list(
+    {"testcase_name": "_dtype={}_mul{}".format(dtype, mul),
+    "dtype": dtype, "mul": mul}
+    for dtype in all_dtypes
+    for mul in [0,1]))
+  def testChebline(self, dtype, mul):
+    rng = jtu.rand_default(self.rng())
+
+    def args_maker():
+      off, scl = rng(2, dtype)
+      scl *= mul
+      return (off, scl)
+
+    jnp_fn = lambda off, scl: cs.chebline(off, scl)
+    np_fn = lambda off, scl: ncs.chebline(off, scl)
```

Comment:
If there are no arguments, you don't need to use an anonymous function. `jnp_fun = cs.chebline` would suffice (or, you can skip the renaming and just use the function directly in the check below)

Same for other tests below.